lib/routes/avatar/get.js

```diff
@@ -33,13 +34,19 @@ module.exports = {
     }
   },
   handler: function avatar(req, reply) {
-    db.getSelectedAvatar(req.auth.credentials.user)
+    var uid = req.auth.credentials.user;
+    db.getSelectedAvatar(uid)
       .then(avatarOrEmpty)
       .done(function (result) {
         var rep = reply(result);
         if (result.id) {
           rep = rep.etag(result.id);
         }
+        var info = {
+          event: 'avatar.get',
+          uid: uid
+        };
+        logger.info('activityEvent', info);
```

Comment:
IIUC this will log an event whenever something *attempts* to read the user's avatar, even if they don't actually have an avatar set (hence the use of `avatarOrEmpty` above).  We could move it into the `if (result.id)` block to ensure it only emits if there's actually an avatar to read.

@vladikoff is it better to measure avatar-reads or avatar-writes for our metrics purposes?